go-client/client/app_stub.go

```diff
@@ -0,0 +1,122 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	apppb2 "github.com/deephaven/deephaven-core/go-client/internal/proto/application"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// Wraps gRPC calls for application.proto.
+type appStub struct {
```

Comment:
Consider renaming to `appServiceStub` or something, since app sounds like an application.